### ISSUE TITLE
Legacy install instructions point to OpenStudio version that isn't compatible

### ISSUE THREAD

**Issue by LyleSY** [ladybug-tools/ladybug-legacy#492]
I am told that the legacy installation requires OpenStudio 2.6 and no newer because it uses an older and compatible version of EnergyPlus. See this post for more on that: https://discourse.ladybug.tools/t/severe-errors-during-energy-simulation-in-honeybee/10160/3

If this is correct, can the install info on the wiki be updated on HoneyBee item 8? https://github.com/ladybug-tools/ladybug-legacy/wiki/Installation-Instructions

**Comment by siown** [comment]
Yes you're right. When I installed OpenStudio 2.8, I got error "1. Failed to download the files needed to run EnergyPlus with OpenStudio 2.x.SystemError(u'\u57fa\u7840\u8fde\u63a5\u5df2\u7ecf\u5173\u95ed: \u53d1\u9001\u65f6\u53d1\u751f\u9519\u8bef\u3002',)". Then I uninstalled OpenStudio 2.8 and tried OpenStudio 2.5, the Honeybee could fly. 
PS: I am working on HB-legacy.

**Comment by mostaphaRoudsari** [comment]
@chriswmackey and @MingboPeng, can you provide some insight here. Should we update the installation instruction?
